kvmagent/kvmagent/test/utils/vm_utils.py

```
@@ -549,6 +600,51 @@ def take_snapshot(vm_uuid, vol_uuid, vol_path, snapshot_path):
 
     return VM_PLUGIN.take_volume_snapshot(misc.make_a_request(cmd.to_dict()))
 
+@misc.return_jsonobject()
+def take_volumes_snapshots(volume_jobs):
+    cmd_body = snapshot_utils.generate_take_volumes_snapshots_default_cmd_body()
+
+    cmd_body["snapshotJobs"].append(snap_job)
```

Ruff (0.12.2)

Undefined name `snap_job`

(F821)